### PR TITLE
{tools}[dummy/dummy] Inspector v2017_update1

### PR DESCRIPTION
**Pull request by shahzebsiddiqui**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/i/Inspector/Inspector-2017_update1.eb [hunk 1]
```diff
@@ -13,4 +13,5 @@ dontcreateinstalldir = 'True'
 
 license_file = HOME + '/licenses/intel/license.lic'
 
+modextrapaths = {'PATH' : ['inspector/bin64'] }
```
Comment (boegel) [inline easybuild/easyconfigs/i/Inspector/Inspector-2017_update1.eb]: @shahzebsiddiqui we need a similar fix for 2016 update3, but with `inspector_xe/bin64`, so we should handle this in the `EB_Inspector` easyblock instead?

Comment (shahzebsiddiqui) [inline easybuild/easyconfigs/i/Inspector/Inspector-2017_update1.eb]: Yea, I think that would be ideal, then we can remove this from the modextrapaths from easyconfig

**Comment by migueldiascosta** [comment]
@shahzebsiddiqui the `EB_Inspector` easyblock does handle this now, doesn't it?

**Comment by boegel** [comment]
I think we can close this, there have been several updates for `Inspector` since this PR was opened, cfr. #6828, #6321, etc.

**Comment by shahzebsiddiqui** [comment]
that is fine this was create a long time ago.

**Pull request closed without merging**